File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/DomainAllInfoAttributesContactSet.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

class DomainAllInfoAttributesContactSet
{
    #[DTA\Data(field: "owner", nullable: true)]
    #[DTA\Strategy("Object", ["type" => \App\DTO\DomainOwnerContact::class])]
    #[DTA\Validator("TypeCompliant", ["type" => \App\DTO\DomainOwnerContact::class])]
    public \App\DTO\DomainOwnerContact|null $owner = null;

    #[DTA\Data(field: "admin", nullable: true)]
    #[DTA\Strategy("Object", ["type" => \App\DTO\DomainAdminContact::class])]
    #[DTA\Validator("TypeCompliant", ["type" => \App\DTO\DomainAdminContact::class])]
    public \App\DTO\DomainAdminContact|null $admin = null;

    #[DTA\Data(field: "tech", nullable: true)]
    #[DTA\Strategy("Object", ["type" => \App\DTO\DomainTechContact::class])]
    #[DTA\Validator("TypeCompliant", ["type" => \App\DTO\DomainTechContact::class])]
    public \App\DTO\DomainTechContact|null $tech = null;

}

File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/DomainAllInfoAttributesContactSet.scala

package org.openapitools.client.model


case class DomainAllInfoAttributesContactSet (
    _owner: Option[DomainOwnerContact],
    _admin: Option[DomainAdminContact],
    _tech: Option[DomainTechContact]
)
object DomainAllInfoAttributesContactSet {
    def toStringBody(var_owner: Object, var_admin: Object, var_tech: Object) =
        s"""
        | {
        | "owner":$var_owner,"admin":$var_admin,"tech":$var_tech
        | }
        """.stripMargin
}
